manta-trusted-setup/Cargo.toml

```diff
@@ -35,24 +35,20 @@ std = []
 test = []
 
 [dependencies]
-ark-ec = { version = "0.3.0", default-features = false }
-ark-ff = { version = "0.3.0", default-features = false }
 ark-groth16 = { version = "0.3.0", default-features = false }
 ark-poly = { version = "0.3.0", default-features = false }
-ark-relations = { version = "0.3.0", default-features = false }
-ark-serialize = { version = "0.3.0", default-features = false, features = ["derive"] }
 ark-std = { version = "0.3.0", default-features = false }
 blake2 = { version = "0.10.4", default-features = false }
 byteorder = { version = "1.4.3", default-features = false }
 derivative = { version = "2.2.0", default-features = false, features = ["use_core"] }
-manta-crypto = { path = "../manta-crypto", default-features = false, features = ["getrandom"] }
+manta-crypto = { path = "../manta-crypto", default-features = false, features = ["arkworks", "getrandom"] }
 manta-util = { path = "../manta-util", default-features = false }
 rand_chacha = { version = "0.3.1", default-features = false }
 workspace-hack = { version = "0.1.0", path = "../workspace-hack" }
 
 [dev-dependencies]
 ark-bls12-381 = { version = "0.3.0", default-features = false, features = ["curve", "scalar_field"] }
-ark-r1cs-std = { version = "0.3.1", default-features = false }
 ark-snark = { version = "0.3.0", default-features = false }
+manta-crypto = { path = "../manta-crypto", default-features = false, features = ["test"] }
```

Comment:
`test` is empty in `manta-crypto`. Why do we need this line if we have Line 44?

Reply:
It's not an empty feature. This is the feature-gate for all the abstract tests including in this case the `pairing` test. I moved it to the `test` feature of `manta-trusted-setup` (using dependency order for inclusion).